cloudflare/import_cloudflare_load_balancer_test.go

```diff
@@ -0,0 +1,44 @@
+package cloudflare
+
+import (
+	"os"
+	"testing"
+
+	"fmt"
+
+	"github.com/cloudflare/cloudflare-go"
+	"github.com/hashicorp/terraform/helper/acctest"
+	"github.com/hashicorp/terraform/helper/resource"
+)
+
+func TestAccCloudFlareLoadBalancer_Import(t *testing.T) {
```

Comment:
Similar error as the other import test:

```console
$ make testacc TESTARGS="-run=TestAccCloudFlareLoadBalancer_Import" 

==> Checking that code complies with gofmt requirements...
TF_ACC=1 go test $(go list ./... |grep -v 'vendor') -v -run=TestAccCloudFlareLoadBalancer_Import -timeout 120m
?       github.com/terraform-providers/terraform-provider-cloudflare    [no test files]
=== RUN   TestAccCloudFlareLoadBalancer_Import
=== PAUSE TestAccCloudFlareLoadBalancer_Import
=== CONT  TestAccCloudFlareLoadBalancer_Import
--- FAIL: TestAccCloudFlareLoadBalancer_Import (0.58s)
        testing.go:434: Step 0 error: Error applying: 1 error(s) occurred:

                * cloudflare_load_balancer_pool.tdu3ryy6ap: 1 error(s) occurred:

                * cloudflare_load_balancer_pool.tdu3ryy6ap: error creating load balancer pool: error from makeRequest: HTTP status 400: content "{\n  \"result\": null,\n  \"success\": false,\n  \"errors\": [\n    {\n      \"code\": 1002,\n      \"message\": \"the origin list length must be in range [1, 0]: validation failed\"\n    }\n  ],\n  \"messages\": []\n}\n"
FAIL
FAIL    github.com/terraform-providers/terraform-provider-cloudflare/cloudflare 0.601s
make: *** [testacc] Error 1
```

Reply:
As above, this is because load balancing is not enabled